### PR TITLE
Stress-ng as aurora task

### PR DESCRIPTION
**Pull request by Damenus**
- Added parser to read metrics form stress-ng
- Added aurora job

### CODE REVIEW

File: owca/wrapper/parser_stress_ng.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+from owca.metrics import Metric, MetricType
+from owca.wrapper import wrapper_main
+from owca.wrapper.parser import readline_with_check
+
+EOF_line = "..."
+
+
+def parse(input: TextIOWrapper, regexp: str, separator: str = None,
+          labels: Dict[str, str] = {}, metric_name_prefix: str = '') -> List[Metric]:
+    """Custom parse function for mutilate
```
Comment (felidadae) [inline owca/wrapper/parser_stress_ng.py]: for mutilate?

**Comment by felidadae** [comment]
I builded the image on 100.64.176.12

File: owca/wrapper/parser_stress_ng.py (hunk 1 above)
Comment (iwankgb) [reply owca/wrapper/parser_stress_ng.py]: @Damenus it is still an issue! :)

Comment (Damenus) [reply owca/wrapper/parser_stress_ng.py]: I changed it in last commit.

**Pull request merged**